resources/views/base.blade.php

```diff
@@ -4,6 +4,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Thom CRUD Tutorial</title>
     <link href="{{ asset('css/app.css') }}" rel="stylesheet" type="text/css" />
+    @yield('css')
```

Comment:
`@yield('css')` để làm gì c nhỉ, nếu muốn tăng tốc độ load thì nên nén css thành min.css chứ. css thì nên đặt chung vào duy nhất 1 file để tái sử dụng, k nên tách ra

Reply:
@hapo-thangbh cái này là t có một đoạn css chỉ muốn dùng trong vài file thôi, không muốn dùng trên toàn hệ thống nên đặt yield vào